internal/learn/about-iota/an-introduction-to-iota.md

```diff
@@ -1,74 +1,15 @@
 ---
 id: an-introduction-to-iota
-title: An Introduction to IOTA
-description: A basic introduction to the IOTA DLT Technology. The use-cases and
-  the Ecosystem developed around it.
+title: IOTA
+description: A basic introduction to the IOTA protocol.
```

Comment:
```suggestion
description: IOTA runs on the Tangle, a structure where newer transactions verify the older ones
```

These articles could do with an `image` tag as well so there's a nice thumbnail if someone shares the link. 

Off the top of my head, it could be a screenshot from one of the gifs, or an IOTA logo.